tutorial_sarscov2.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: default
+permalink: /tutorial/sars-cov2/
+---
+
+This tutorial shows you the basic of how to interact with V-pipe.
+
+For the purose of this Tutorial, we will work with the `sars-cov2` branch which is [adapted for the SARS-CoV-2 virus]({{ "/sars-cov-2/" | relative_url }}).
+
+> **Organisation of Data** :
+>
+> V-pipe expects its samples data organized in a two-level hierarchy:
+>
+> - Input files to be grouped by samples (e.g.: patient samples or biological replicates of an experiment).
```

Comment:
If you start the second point with 'A second level', it would be intuitive to start the first point with something like 'A/At a first level, ...'

Reply:
I am trying to reformulate as you suggest.